protocol/communities/persistence.go

```diff
@@ -828,6 +828,10 @@ func (p *Persistence) DeclinedPendingRequestsToJoinForCommunity(id []byte) ([]*R
 	return p.RequestsToJoinForCommunityWithState(id, RequestToJoinStateDeclinedPending)
 }
 
+func (p *Persistence) OwnershipChangedRequestsToJoinForCommunity(id []byte) ([]*RequestToJoin, error) {
```

Comment:
```suggestion
func (p *Persistence) AwaitingAddressesRequestsToJoinForCommunity(id []byte) ([]*RequestToJoin, error) {
```

Reply:
Small variation on Patryk's idea to make it sound better
```suggestion
func (p *Persistence) RequestsToJoinForCommunityAwaitingAddresses(id []byte) ([]*RequestToJoin, error) {
```